code/library/src/main/kotlin/org/fbme/lib/st/expressions/BinaryOperation.kt

```diff
@@ -1,10 +1,10 @@
 package org.fbme.lib.st.expressions
 
-enum class BinaryOperation(val alias: String) {
-    ADD("+"), AMP("&"), AND("AND"), DIV("/"), EQ("="), GT(">"), GTE(">="), LT("<"), LTE("<="), MOD("MOD"), MUL("*"), NEQ(
-        "<>"
-    ),
-    OR("OR"), POW("**"), SUB("-"), XOR("XOR");
+enum class BinaryOperation(val alias: String, val luaAlias: String) {
```

Comment:
I would extract `luaAlias` from this definition -- in general `library` code shall be not aware of specific runtimes.

Reply:
fixed